src/actions/ui.js

```diff
@@ -65,6 +65,12 @@ export const startEditingInstructions = createAction(
   (_projectKey, timestamp = Date.now()) => ({timestamp}),
 );
 
+export const continueEditingInstructions = createAction(
```

Comment:
This is of course the camel case version of the action name, and for the action I was following the pattern of the actions `{verb}_EDITING_INSTRUCTIONS`. Other verbs are `start` and `cancel`. You would rather break that pattern? It just seems a bit less intuitive but I'm fine with it. So it would be `UPDATE_IN_PROGRESS_INSTRUCTIONS`? I figure `continueEditingInstructions` fits in nicely with `startEditingInstructions` and `cancelEditingInstructions`. Another idea for something more verbose is `continueEditingUnsavedInstructions`.

Reply:
Definitely understand the appeal of the symmetry, but ultimately I think clarity/expressiveness is the overriding concern here. Given that you went with `draftInstructions` for the property name (which I like) I would propose `updateDraftInstructions` here. WDYT?